examples/quickstart/sort_by_key_demo.cpp

```diff
@@ -51,6 +51,5 @@ int pika_main()
 ///////////////////////////////////////////////////////////////////////////////
 int main(int argc, char* argv[])
 {
-    return pika::init(
-        pika_main, argc, argv);    // Initialize and run pika
+    return pika::init(pika_main, argc, argv);    // Initialize and run pika
```

Comment:
Unrelated to this PR, but we can probably have the clang-format check enabled for all source files now (including .cu files), right?

Reply:
And related to clang-format, I wouldn't mind bumping the clang version if needed.

Reply:
I opened #168